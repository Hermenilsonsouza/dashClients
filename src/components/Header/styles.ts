import styled from 'styled-components'

export const Wrapper = styled.main`
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: #202124;
  padding: 16px;
`

export const Title = styled.h1`
  color: #fafafa;
  font-size: 28px;
`
