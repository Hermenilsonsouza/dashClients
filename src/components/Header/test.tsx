import { render, screen } from '@testing-library/react'

import Header from '.'

describe('<Header />', () => {
  it('should render the heading', () => {
    const { container } = render(<Header title="Lista de Clientes" />)

    expect(
      screen.getByRole('heading', { name: /Lista de Clientes/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
