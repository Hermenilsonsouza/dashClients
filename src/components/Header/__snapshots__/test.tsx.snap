// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render the heading 1`] = `
.c0 {
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: #202124;
  padding: 16px;
}

.c1 {
  color: #fafafa;
  font-size: 28px;
}

<main
  class="c0"
>
  <h1
    class="c1"
  >
    Lista de Clientes
  </h1>
</main>
`;
